src/main/scala/org/hammerlab/guacamole/distributed/TaskPosition.scala

```diff
@@ -1,19 +1,23 @@
 package org.hammerlab.guacamole.distributed
 
+import org.hammerlab.guacamole.distributed.LociPartitionUtils.PartitionIndex
+
 /**
  * TaskPosition represents the task a read is assigned to and the start position on the reference genome of the read
  * Each read is assigned to a task and the reads are sorted by (referenceContig, locus) when they are processed
  *
- * @param task Task ID
+ * @param partition Task ID
```

Comment:
Yea I think some of #496 makes sense to just fold in to other PRs like this one, which I thought you sort of alluded to there.